components/image/slider/src/index.js

```diff
@@ -128,9 +132,9 @@ export default function ImageSlider({
     slides.length > 0 && (
       <div onClick={handleClick} className={BASE_CLASS}>
         {slides.length > 1 ? (
-          <ReactSlidy {...sliderOptions} doAfterSlide={handleAfterSlide}>
+          <MoleculeCarousel {...sliderOptions} doAfterSlide={handleAfterSlide}>
```

Comment:
```suggestion
          <MoleculeCarousel {...sliderOptions} onSlideAfter={handleAfterSlide}>
```

Reply:
maybe there are other being used in the vertical...

Reply:
you're right, I should've read the migration docs!

just did the changes and tagged them as breaking changes
https://github.com/SUI-Components/adevinta-spain-components/pull/524/commits/e4e7ed6b0849af4278eb2e91904f3b533af4cd23